src/startupscriptgenerator/dotnetcore/main.go

```diff
@@ -31,7 +32,13 @@ func main() {
 	defaultAppFilePathPtr := flag.String(
 		"defaultAppFilePath",
 		"",
-		"[Optional] Path to a default dll that will be executed if the entrypoint is not found. Ex: '/opt/startup/aspnetcoredefaultapp.dll'")
+		"[Optional] Path to a default dll that will be executed if the entrypoint is not found." +
+		" Ex: '/opt/startup/aspnetcoredefaultapp.dll'")
+	copyOutputToDifferentDirAndRunPtr := flag.String(
+		"copyOutputToDifferentDirAndRun",
+		"true",
```

Comment:
I made this `true` the default just to make sure all the tests pass under this flag.